odalic/pom.xml

```diff
@@ -39,6 +39,10 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</repository>
+		<repository>
+			<id>stardog-public</id>
```

Comment:
The Pinto library https://github.com/stardog-union/pinto , which generates the RDF model based on class annotations, depends on some of them (e.g. those with groupId com.complexible.common: cp-openrdf-utils and cp-common-utils), as the Pinto is from the same company.